rust/core-lib/src/tabs.rs

```diff
@@ -239,18 +263,18 @@ impl<W: Write + Send + 'static> Tabs<W> {
 }
 
 impl<W: Write> TabCtx<W> {
-    pub fn update_view(&self, view_id: &str, update: &Value) {
+    pub fn update_view(&self, view_id: &ViewIdentifier, update: &Value) {
         self.rpc_peer.send_rpc_notification("update",
             &json!({
-                "view_id": view_id,
+                "view_id": view_id.clone().into_value(),
```

Comment:
Yeah, will do. Was just slopping it together and didnt want to look at docs for implementing Serialize/Deserialize manually